src/_data/team.yml

```diff
@@ -308,12 +308,14 @@
   image: /assets/custom/img/team/mattsi_jansky.jpg
   role: Software Craftsman
   fullDescription:
-    <p>Mattsi is an ambitious software developer that loves to learn and belives in "doing it right the first time", prioritising quality software over risky shortcuts.</p>
-    <p>Before joining Codurance he experienced a broad career across full-stack and back-end development roles in education, international e-commerce and financial technology focusing on the ASP.Net stack.</p>
+    <p>Mattsi is an ambitious software developer that loves to learn and belives in "doing it right the first time", putting quality software first.</p>
```

Comment:
@Mattsi-Jansky There is a typo in the `belives` word.

Reply:
Oh man, I can't believe no-one's noticed that until now 😆 Thanks!

Reply:
argh :(